src/components/form/Input/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export default function index({ name, type, placeholder, ref, label }) {
```

Comment:
so...what does the type param mean

Reply:
imagine you are me using this component. write a comment

Reply:
it is the same as `type` attribute of `<input>` exactly all of these params. The point of that sub-component is to wrap input with all these styles, error message, and so on.

Reply:
done!